python_modules/dagster-graphql/dagster_graphql_tests/client_tests/sleepy.py

```diff
@@ -54,23 +51,10 @@ def total(_, in_1, in_2, in_3, in_4):
     return in_1 + in_2 + in_3 + in_4
 
 
-@pipeline(
-    description=(
-        "Demo diamond-shaped pipeline that has four-path parallel structure of solids.  Execute "
-        "with the `multi` preset to take advantage of multi-process parallelism."
-    ),
-    preset_defs=[
-        PresetDefinition(
-            "multi",
-            {
-                "execution": {"multiprocess": {}},
-                "solids": {"giver": {"config": [2, 2, 2, 2]}},
-            },
-        )
-    ],
-    mode_defs=[default_mode_def_for_test],
+@job(
+    description="Demo diamond-shaped job that has four-path parallel structure of ops.",
```

Comment:
should maybe specify a config arg for that op config?

Reply:
ok